backend/packages/Upgrade/src/api/models/ArchivedStats.ts

```diff
@@ -0,0 +1,18 @@
+import { Entity, PrimaryColumn, Column, OneToOne, JoinColumn } from 'typeorm';
+import { IsNotEmpty } from 'class-validator';
+import { BaseModel } from './base/BaseModel';
+import { Query } from './Query';
+
+@Entity()
+export class ArchivedStats extends BaseModel {
+  @PrimaryColumn('uuid')
+  public id: string;
+
+  @IsNotEmpty()
+  @Column('jsonb')
+  public result: any;
```

Comment:
sry gotta be annoying about `any`, is there any type that can be defined for this? What does "result" mean?

Reply:
the result stores the query result in a JSON object. Will update this to appropriate type

Reply:
code updated